CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 - Debug from .csproj and .sln [#5876](https://github.com/dotnet/vscode-csharp/issues/5876)
 
 # Latest
+* Update Roslyn to 4.12.0-3.24430.2 (PR: [#7496](https://github.com/dotnet/vscode-csharp/pull/7496))
```

Comment:
Does this need more description?

Reply:
There's not really anything super relevant to VSCode features in the changelog (just a couple of pre-requisite changes for other things that aren't ready yet).